Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php

```diff
@@ -61,9 +61,48 @@
             'cols' => 30,
             'rows' => 4,
         ],
-    ],
+    ]
 ];
 
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addTCAcolumns(
+    'tx_staffdirectory_domain_model_organization',
+    [
+        'tx_staffdirectoryorganization_image' => [
+            'exclude' => true,
+            'label' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.images',
+            'config' => \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::getFileFieldTCAConfig(
+                'tx_staffdirectoryorganization_image',
+                [
+                    'appearance' => [
+                        'createNewRelationLinkTitle' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.images',
+                        'fileUploadAllowed' => true,
+                        'maxitems' => 5,
+                    ],
+                    'overrideChildTca' => [
+                        'types' => [
+                            \TYPO3\CMS\Core\Resource\File::FILETYPE_IMAGE => [
+                                'showitem' => '
+                                    --palette--;;imageoverlayPalette,
+                                    --palette--;;filePalette
+                                ',
+                            ],
+                        ],
+                    ],
+                ],
+                $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext']
+            ),
+        ],
+    ]
+);
+
```

Comment:
Why don't you add this to `$tempColumns` instead?

Reply:
I wanted to put the image tab not in contact tab.